vagga.yaml

```diff
@@ -51,6 +51,18 @@ containers:
       /run: !Tmpfs
         subdirs:
           postgresql: { mode: 0o777 }
+  test:
+    setup:
+      - !Ubuntu xenial
```

Comment:
I tried some variants. This is not depends on version of alpine. First time I thought that maybe v3.4 of Alpine may be a bit deprecated - after I tried it with v3.5 - result was the same.I also installed it under Ubuntu trusty  and it works. 
I can be wrong but I think Ubuntu has some packages that does not have Alpine, that's why we have trouble with it...